src/server/verification/api/__tests__/__util__/index.js

```diff
@@ -65,6 +66,21 @@ export default zoomServiceMock => {
       }
     })
 
+  const mockLivenessError = () =>
+    zoomServiceMock.onPost('/search').reply(400, {
+      meta: {
+        code: 400,
+        ok: false,
+        m: { zb: 2256, st: 1594565626.252231, bt: null, bp: null, zw: 2545, zp: 2528 },
```

Comment:
could be removed - it's some optional data non described in the API docs. our exception handler doesn't looks on it